### PR TITLE
Fix CI failing with `xgboost==1.6.0`

### PR DESCRIPTION
**Pull request by Yard1**

### CODE REVIEW

File: xgboost_ray/tests/test_sklearn.py [hunk 1]
```diff
@@ -313,7 +313,7 @@ def test_num_parallel_tree(self):
             dump = bst.get_booster().get_dump(dump_format="json")
             assert len(dump) == 4
 
-            if XGBOOST_LOOSE_VERSION >= LooseVersion("2.0.0"):
+            if XGBOOST_LOOSE_VERSION >= LooseVersion("1.6.0"):
                 config = json.loads(bst.get_booster().save_config())
                 assert (int(config["learner"]["gradient_booster"][
                     "gbtree_model_param"]["num_parallel_tree"]) == 4)
```
Comment (trivialfis) [inline xgboost_ray/tests/test_sklearn.py:319]: Out of curiosity, would it be useful if we expose this as a property?

Comment (Yard1) [reply xgboost_ray/tests/test_sklearn.py:319]: `XGBOOST_LOOSE_VERSION` or `"1.6.0"`?

Comment (krfricke) [reply xgboost_ray/tests/test_sklearn.py:319]: I think @trivialfis means the number of trees. Yes, that would actually be very helpful, we have a few tests that use this

**Review by krfricke — APPROVED** [review]

File: xgboost_ray/tests/test_sklearn.py (hunk 1 above)
Comment (Yard1) [reply xgboost_ray/tests/test_sklearn.py:319]: Ah, gotcha.

**Pull request merged**